### DISCUSSION TITLE
Store clips within a separate "clips" folder within each channel's folder

### DISCUSSION THREAD

**Discussion by jayjay181818** [Zibbp/ganymede#559] [Ideas]
I imagine my use case is a bit niche and this won't be an issue for 99% of Ganymede users, but would it be possible to have clips save into a separate folder such as "clips" within each channel's directory? I only ask since I have a an rsync job setup that runs to me and a creator's Synology nas for quicker access to VODS. But now with the new clip visibility functionality added in today's update, clip archiving will be something I will be enabling. The only issue is that this will make the organisation of the folder a lot harder, given there will be a lot more directories due to clips now being included.

**Comment by Zibbp** [comment]
Unfortunately that is not possible and I don't plan on supporting a setup like that. Clips are just a different type of video so it runs through the standard processing flow. Clips in video directories would also present issues for clip video's that aren't archived or are deleted. 

The default directory template name includes `clip` in the name, so maybe that can be used to filter on your end? Additionally the `/api/v1/vod/<VIDEO_ID>/clips` API endpoint will return clips for a video.
